debian/copyright

```diff
@@ -1,10 +1,20 @@
 Format: http://dep.debian.net/deps/dep5
 Upstream-Name: nest-simulator
-Source: https://github.com/nest/nest-simulator/archive/master.zip
+Source: https://github.com/nest/nest-simulator/
+
+Files: *
+Copyright: 2004-2019 The NEST Initiative <nest_user@nest-initiative.org>
+License: GPL-2+
```

Comment:
Why do you write` GPL-2+` and not `GPL-2 `? We use the GPL version 2.0, so I think `GPL-2` is correct.

Reply:
After having read the license again I agree with GPL-2+. Sorry.